### PR TITLE
Functional tests - Update exchange rate

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Update exchange rate
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/11_international/01_localization/currencies/04_updateExchangeRate.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20539)

### CODE REVIEW

File: tests/UI/pages/BO/international/currencies/index.js [hunk 1]
```diff
@@ -183,6 +187,26 @@ class Currencies extends LocalizationBasePage {
...
+  async goToEditCurrencyPage(page, row = 1) {
+    await this.clickAndWaitForNavigation(page, this.editRowLink(row));
+  }
+
+  /**
+   * Click on update exchange rates
+   * @param page
+   * @returns {Promise<string>}
+   */
+  async clickUpdateExchangeRates(page) {
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/international/currencies/index.js]:
```suggestion
  async updateExchangeRates(page) {
```

File: tests/UI/campaigns/functional/BO/11_international/01_localization/currencies/04_updateExchangeRate.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      // Update and check successful message
+      const textResult = await addCurrencyPage.updateExchangeRate(page, newExchangeRate);
+      await expect(textResult).to.contains(currenciesPage.successfulUpdateMessage);
+    });
+
+    it('should click on Update exchange rate button', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickUpdateExchangeRates', baseContext);
+
+      // Click update exchange rates button and check successful message
+      const textResult = await currenciesPage.clickUpdateExchangeRates(page);
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/11_international/01_localization/currencies/04_updateExchangeRate.js]:
```suggestion
      // Click on update exchange rates button and check successful message
      const textResult = await currenciesPage.updateExchangeRates(page);
```

File: tests/UI/pages/BO/international/currencies/index.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/international/currencies/index.js]: Done thx :)

File: tests/UI/campaigns/functional/BO/11_international/01_localization/currencies/04_updateExchangeRate.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/11_international/01_localization/currencies/04_updateExchangeRate.js]: Done thx :)

**Review by boubkerbribri — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**